### PR TITLE
View/UI

### PR DESCRIPTION
**Pull request by pszi**
Since the source packages have been restructured recently, I'm not sure that the files could be transited into their right places, can you please help with that? Also, I don't mind if the merge is postponed, but perhaps it would be a good idea if every new branch could come off of main from now on.

### CODE REVIEW

**Comment by vismate** [comment]
I mean we could do a manual merge, but I think it would be way easier for now if you just made the same restucturing we did on main. (Resolving conflicts manually would be essentially the same.) After that, we'll test an review the code. 😁

**Comment by vismate** [comment]
Just looked at the code, looks good. I'll build and test it in a few hours. It has conflicts with the camera control branch already merged into main, so we'll need to resolve the conflicts. I propose we all review the code and make necessary changes this week. At our weekly meeting we could go over it together to resolve the conflicts.

**Comment by pszi** [comment]
The the source of that conflict is simply that I've changed it so that dragging the mouse would only spam the coordinate values in the console when the right button is being pressed, because it interfered with my manual testing of the menu buttons. It was literally moving one if-condition.
How does reviewing these conflicts work? Is it just the work of one person, or do we need a consensus? I don't want to accidentally remove someone else's code.

File: app/src/main/java/ELME/View/Toolbar.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package View;
```
Comment (vismate) [inline app/src/main/java/ELME/View/Toolbar.java]: Incorrect package. Should be ELME.View

File: app/src/main/java/ELME/View/SideMenu.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
+package View;
```
Comment (vismate) [inline app/src/main/java/ELME/View/SideMenu.java]: Incorrect package

File: app/src/main/java/ELME/View/SideMenu.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+import de.gurkenlabs.litiengine.Game;
+import de.gurkenlabs.litiengine.gui.ImageComponent;
+import de.gurkenlabs.litiengine.input.Input;
+
+import java.awt.Color;
+import java.awt.event.MouseEvent;
+import java.util.List;
```
Comment (vismate) [inline app/src/main/java/ELME/View/SideMenu.java]: Unused imports from here on

File: app/src/main/java/ELME/View/SideMenu.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import java.util.function.Consumer;
+
+/**
+ * @author pszi
+ */
+public class SideMenu extends ExtraMenu {
+
+    public static final Color SIDE_FOREGROUND = new Color(100,180,120, 175);
+    public static final Color SIDE_BACKGROUND = new Color(100,100,120, 175);
+    //private final List<Consumer<Integer>> confirmConsumer;
```
Comment (vismate) [inline app/src/main/java/ELME/View/SideMenu.java]: Commented lines could be removed.

File: app/src/main/java/ELME/View/ExtraMenu.java [hunk 5]
```diff
@@ -0,0 +1,81 @@
+package View;
```
Comment (vismate) [inline app/src/main/java/ELME/View/ExtraMenu.java]: Incorrect package

**Review by vismate — CHANGES_REQUESTED** [review]

**Review by vismate — APPROVED** [review]

**Pull request merged**